invenio_drafts_resources/services/records/service.py

```diff
@@ -161,7 +161,7 @@ def update_draft(self, id_, identity, data, revision_id=None):
         for component in self.components:
             if hasattr(component, 'update_draft'):
                 component.update_draft(
-                    identity, record=draft, data=data)
+                    identity, record=draft, data=data, errors=errors)
```

Comment:
Comment: The API change makes sense, however it's likely that there's other components that have to be fixed so just perhaps git grep for ``update_draft()`` in modules to check

Reply:
Double checked that, for example `RDMRecordServiceConfig` have in addition to `DraftFilesComponent`:
- MetadataComponent
- AccessComponent
- PIDComponent

all of them have `update_draft` method defined in such a way:
```
def update_draft(self, identity, data=None, record=None, **kwargs)
```
So I guess it's ok